mbed_build/mbed_tools.py

```diff
@@ -3,6 +3,8 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 """Integration with https://github.com/ARMmbed/mbed-tools."""
+from mbed_build._internal.mbed_tools.config import config
 from mbed_build._internal.mbed_tools.export import export
 
-cli = export
```

Comment:
I suspect this is going to make `mbed-tools` unhappy? If they're not locked, maybe it's worth retaining `cli` export for now additionally to `config` and `export` to avoid breakage?

Reply:
Yeah, I could make it a softer transition 👍

Reply:
05f8974